src/components/ui/form.stories.tsx

```
@@ -0,0 +1,83 @@
+			submitCount: 1,
+			dirtyFields: fn(),
+			touchedFields: fn(),
+			validatingFields: fn(),
+			errors: {},
+		},
+		resetField: fn(),
+		reset: fn(),
+		handleSubmit: fn(),
+		unregister: fn(),
+		// eslint-disable-next-line
+		control: fn() as any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)